components/image/graphviz/Makefile

```diff
@@ -40,15 +40,19 @@ include $(WS_TOP)/make-rules/prep.mk
 include $(WS_TOP)/make-rules/configure.mk
 include $(WS_TOP)/make-rules/ips.mk
 
-PATH =		/usr/bin:/usr/gnu/bin:/usr/php/5.6/bin
+PATH_PHP.32 = /usr/php/5.6/bin 
```

Comment:
If we dedice to go for it, please do not add it to shared-macros.mk, but create a file specific to PHP and include it in shared-macros.mk